worldengine/generation.py

```diff
@@ -200,7 +200,7 @@ def generate_world(w, step):
 
     # Prepare sufficient seeds for the different steps of the generation
     rng = numpy.random.RandomState(w.seed)  # create a fresh RNG in case the global RNG is compromised (i.e. has been queried an indefinite amount of times before generate_world() was called)
-    sub_seeds = rng.randint(0, 4294967295, size=100)  # sys.maxsize didn't quite work
+    sub_seeds = rng.randint(0, 2147483647, size=100)  # sys.maxsize didn't quite work
```

Comment:
I got that error message too, but only for uint64. This is only a problem for numpy, not python itself. 

I find it odd that people install 32-bit python on 64-bit windows. If you had used 64-bit python, this wouldn't be a problem. In addition to this, when we go to make official releases, it will be 64-bits only.